rfc9204.md

```diff
@@ -928,9 +911,9 @@ The encoder transforms the Required Insert Count as follows before encoding:
 
 Here `MaxEntries` is the maximum number of entries that the dynamic table can
```

Comment:
On lines 912-920, the RFC Editor text output does not have quotes, but (as with the HTTP/3 AUTH48) this appears to be a tooling difference rather than a deliberate change.  The XML matches.

Reply:
Note that back ticks result in a change to monospace font in HTML but no other decoration. This is a recent change in xml2rfc, so you might be seeing changes in the diff.